rllib/algorithms/algorithm_config.py

```diff
@@ -3045,6 +3047,8 @@ def get_marl_module_spec(
                     module_specs={
                         k: copy.deepcopy(single_agent_spec) for k in policy_dict.keys()
                     },
+                    load_state_path=cur_marl_module_spec.load_state_path,
+                    load_state_ip_addr=cur_marl_module_spec.load_state_ip_addr,
```

Comment:
how is load_state_ip_addr saved and used?
it's unlikely this will be useful across runs right?

Reply:
it won't be useful across runs, no